projects/fishtornado_xr/FishTornado.cpp

```diff
@@ -493,13 +493,14 @@ void FishTornadoApp::UpdateTime()
 void FishTornadoApp::UpdateScene(uint32_t frameIndex)
 {
     PerFrame& frame = mPerFrame[frameIndex];
-
```

Comment:
probably better as a separate function in case other part of the code need to access the camera